db/migrate/20210427162657_add_player_counts_to_posts.rb

```diff
@@ -0,0 +1,6 @@
+class AddPlayerCountsToPosts < ActiveRecord::Migration[6.1]
+  def change
+    add_column :posts, :min_players, :integer, default: 1
```

Comment:
Just adding this as a reminder that this should default to `null` or whatever so old posts don't show up in every result even though they might not be relevant